__tests__/app.test.js

```diff
@@ -92,4 +92,48 @@ describe('app.js', () => {
             })
         })
     })
+
+    describe('GET /api/articles', () => {
+        test('200: responds with all articles', () => {
+            return request(app).get('/api/articles')
```

Comment:
In this test you're using a combination of `hasOwnProperty` and `typeof`, because the `typeof` assertions rely on the property existing ( otherwise they would be undefined for the type ) you don't need the `hasOwnProperty` as it's all covered by the `typeof`